taxi/urls.py

```diff
@@ -7,20 +7,44 @@
     DriverListView,
     DriverDetailView,
     ManufacturerListView,
+    CarCreateView,
+    ManufacturerCreateView,
+    CarUpdateView,
+    CarDeleteView,
+    ManufacturerUpdateView,
+    ManufacturerDeleteView,
 )
 
 urlpatterns = [
     path("", index, name="index"),
+    path("cars/", CarListView.as_view(), name="car-list"),
+    path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
+    path("cars/<int:pk>/update", CarUpdateView.as_view(), name="car-update"),
+    path("cars/<int:pk>/delete", CarDeleteView.as_view(), name="car-delete"),
```

Comment:
Add a trailing `/` to all your endpoints.

Reply:
do it